QUARKUS-1415.md

```diff
@@ -0,0 +1,30 @@
+# QUARKUS-1415 Reactive REST Client improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1415
+
+Test plan covers Reactive REST Client functionality provided by `quarkus-rest-client-reactive` extension.
+
+Documentation:
+https://quarkus.io/version/main/guides/resteasy-reactive
+https://quarkus.io/version/main/guides/rest-client-reactive
+
+## Related test plans
+- [QUARKUS-1075](QUARKUS-1075.md)
+
+## Scope of the testing
+- Upload and download of large(more than 2Gb) files, see https://github.com/quarkusio/quarkus/pull/22229 for details
```

Comment:
Is it realistic to fetch 2 GB of data using REST Client?

Reply:
It is, but it will take some time(between 10 and 30 minutes, I guess)

Reply:
```suggestion
- Upload and download of large (more than 2GB) files, see https://github.com/quarkusio/quarkus/pull/22229 for details
```

let's see if native will be able to handle 2GB+ size :)